README.md

```diff
@@ -15,6 +15,19 @@ have [a detailed introductory video][], available for free on Upcase.
 
 [a detailed introductory video]: https://upcase.com/videos/factory-girl?utm_source=github&utm_medium=open-source&utm_campaign=factory-girl
 
+Project History and Naming
+--------------------------
+
+The name "Factory Girl" might be confusing to some developers who encounter this library. The name [was chosen](https://robots.thoughtbot.com/waiting-for-a-factory-girl) as a clever (or so we hoped) reference to the [Rolling Stones song](https://www.youtube.com/watch?v=4jKix2DFlnA) of the same name, and as a nod in the direction of the [Factory method pattern](https://en.wikipedia.org/wiki/Factory_method_pattern) from the Design Patterns book. Our intentions were somewhere between amusing and harmless, and any meaning beyond that which can be attached to the project name is unintentional.
```

Comment:
"Our intentions were somewhere between amusing and harmless, and " - delete, repeats "he name [was chosen](https://robots.thoughtbot.com/waiting-for-a-factory-girl) as a clever"